src/neo4j_genai/generation/prompts.py

```diff
@@ -28,8 +35,8 @@ class PromptTemplate:
     missing, a `PromptMissingInputError` is raised.
     """
 
-    DEFAULT_TEMPLATE: str = ""
-    EXPECTED_INPUTS: list[str] = []
+    DEFAULT_TEMPLATE: str = "{query_text}"
```

Comment:
I'm not sure we need these defaults since we are testing against all the items in `expected_inputs`, do we?

Reply:
you're right, tests still pass when I remove the `query_text` and use the original empty defaults